test/int_64_vm_test.dart

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2012, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+@TestOn('!js')
+
+import 'package:fixnum/fixnum.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group("conversions", () {
+    test("toInt", () {
+      expect(Int64.parseInt("-10000000000000000").toInt(),
+          same(-10000000000000000));
+      expect(Int64.parseInt("-10000000000000001").toInt(),
+          same(-10000000000000001));
+      expect(Int64.parseInt("-10000000000000002").toInt(),
+          same(-10000000000000002));
+      expect(Int64.parseInt("-10000000000000003").toInt(),
+          same(-10000000000000003));
+      expect(Int64.parseInt("-10000000000000004").toInt(),
+          same(-10000000000000004));
+      expect(Int64.parseInt("-10000000000000005").toInt(),
+          same(-10000000000000005));
+      expect(Int64.parseInt("-10000000000000006").toInt(),
+          same(-10000000000000006));
+      expect(Int64.parseInt("-10000000000000007").toInt(),
+          same(-10000000000000007));
+      expect(Int64.parseInt("-10000000000000008").toInt(),
+          same(-10000000000000008));
+    });
+  });
+
+  test("", () {
```

Comment:
Why doesn't this have a name?

Reply:
Agreed – weird. It didn't have a name in the other file, either. Just keeping this consistent.